Source/CountDown.spoon/init.lua

```diff
@@ -90,42 +280,92 @@ end
 ---  * None
 
 function obj:pauseOrResume()
-    if obj.timer then
-        if obj.timer:running() then
-            obj.timer:stop()
-        else
-            obj.timer:start()
-        end
+   -- if the timer is paused, we need to offset
+   -- the starting time for as long as the timer is paused
+
+   if obj.timer and obj.timerBar then
+      if obj.timer:running() then
+         obj.pausedAt = obj:time_absolute_seconds()
+         obj.timer:stop()
+         obj.timerBar:stop()
+
+         obj:menu_items(true)
+
+      else
+         -- offset the starting time by as many seconds as we were paused
+         local pausedSeconds = obj:time_absolute_seconds() - obj.pausedAt
+         obj.startTime = obj.startTime + pausedSeconds
```

Comment:
I would think this calculation is not valid if there are multiple pauses and resumes.